netbout-client/src/test/java/com/netbout/mock/MkInboxTest.java

```diff
@@ -46,12 +50,25 @@
      */
     @Test
     public final void testStart() throws IOException {
-        final String alias = "current-name";
-        final MkInbox inbox = new MkInbox(new H2Sql(), alias);
+        final String name = "current-name";
+        final Sql sql = new H2Sql();
+        final Aliases aliases = new MkUser(
```

Comment:
```
  @Test
    public final void testStart() throws IOException {
        final MkBase base = new MkBase();
        final String name = base.randomAlias().name();
        final Bout bout = base.randomBout();
        bout.friends().invite(name);
        MatcherAssert.assertThat(
            bout.friends().iterate(),
            Matchers.hasItem(new Friend.HasAlias(Matchers.is(name)))
        );
    }
```

@pinaf i can refactor original test like this but i am not sure it is correct. Because current test is testing <code>inbox.start</code> directly, refactored version will test it similar way but not same way.

```
  @Test
    public final void testStart() throws IOException {
        final MkBase base = new MkBase();
        final Bout bout = base.randomBout();
        MatcherAssert.assertThat(
            bout.friends().iterate(),
            Matchers.hasItem(
                bout.friends().iterate().iterator().next()   
            )
        );
    }
```

or maybe like this. What do you think ?

Original test is testing user is invites <b>self</b> . Open suggestions.